version/constants.go

```diff
@@ -60,17 +60,17 @@ var (
 	// by avalanchego, but is useful for downstream libraries.
 	RPCChainVMProtocolCompatibility map[uint][]*Semantic
 
+	DefaultTime = time.Date(2020, time.December, 5, 5, 0, 0, 0, time.UTC)
```

Comment:
```suggestion
	DefaultUpgradeTime = time.Date(2020, time.December, 5, 5, 0, 0, 0, time.UTC)
```
Feels a bit more descriptive